controls/os_spec.rb

```diff
@@ -21,7 +21,7 @@
 login_defs_passmaxdays = attribute('login_defs_passmaxdays', default: '60', description: 'Default password maxdays to set in login.defs')
 login_defs_passmindays = attribute('login_defs_passmindays', default: '7', description: 'Default password mindays to set in login.defs')
 login_defs_passwarnage = attribute('login_defs_passwarnage', default: '7', description: 'Default password warnage (days) to set in login.defs')
-if os.redhat?
+if os.redhat? || os.suse?
   shadow_group = 'root'
 elsif os.debian?
   shadow_group = 'shadow'
```

Comment:
@chris-rock As I had to rework my patch I already implemented the second option ...